pallets/phala/src/stakepool.rs

```diff
@@ -2940,7 +2953,7 @@ pub mod pallet {
 				set_block_1();
 				setup_workers(1);
 				setup_pool_with_workers(1, &[1]); // pid=0
-				// Start a worker as usual
+								  // Start a worker as usual
```

Comment:
intention

Reply:
`cargo fmt` automatically changed it to this. We should create another PR to fix the format config